src/routes/clients.js

```diff
@@ -52,4 +53,13 @@ const validatorGetAllClients = Joi.object({
 });
 router.get('/', validator.query(validatorGetAllClients), wrapper(getAllClients));
 
+const validatorVisitsClientID = Joi.object({

```

Comment:
You've got this `validatorClientID`. Use it, please.

Reply:
`validatorClientID` has `id` parameter, but I  need to validate `clientID` parameter. To use `validatorClientID` I should be getting '/:id/visits' endpoint, but I am getting '/:clientID/visits'

Reply:
Ah, I got it.